### PR TITLE
Updated github actions used to perform releases

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Changed the build/release process to be similar to the one used by conda-lock

### CODE REVIEW

File: ensureconda/__init__.py [hunk 1]
```diff
@@ -1,4 +1,9 @@
-__version__ = "1.3.0"
-
+import pkg_resources
 
 from .api import ensureconda
+
+
+try:
+    __version__ = pkg_resources.get_distribution("conda_lock").version
```
Comment (ocefpaf) [inline ensureconda/__init__.py:7]: Ideally we should move away from `pkg_resources` but this is fine for now/

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      - name: Test wheels
+        run: |
+          python -m twine check dist/*
+        shell: bash
+
+      - name: Publish a Python distribution to PyPI
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_PASSWORD }}
```
Comment (ocefpaf) [inline .github/workflows/release.yml:41]: Did you add the secret to the repo or do you want me to do so?

**Review by ocefpaf — APPROVED** [review]

File: ensureconda/__init__.py (hunk 1 above)
Comment (mariusvniekerk) [reply ensureconda/__init__.py]: Spotted this should be `ensureconda`

File: .github/workflows/release.yml (hunk 2 above)
Comment (mariusvniekerk) [reply .github/workflows/release.yml:41]: I've added it already

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,70 @@
+[tool:pytest]
+flake8-max-line-length = 105
+flake8-ignore =
+    docs/* ALL
+    conda_lock/_version.py ALL
```
Comment (ocefpaf) [inline setup.cfg:5]: ensurecoda

**Pull request merged**